conf/st2.conf.sample

```diff
@@ -155,10 +151,22 @@ ssl_certfile = None
 ssl_keyfile = None
 # If True and `ssl_cert_reqs` is not None, enables hostname verification
 ssl_match_hostname = True
+# Create the connection to mongodb using TLS.
+tls = False
+# Specifies whether MongoDB is allowed to pass an invalid certificate. This defaults to False to have security by default. Only temporarily set to True if you need to debug the connection.
+tlsAllowInvalidCertificates = None
```

Comment:
I prefer remaining consistent with the rest of the `st2.conf` variable naming convention and use snake case as opposed to MongoDB's camel case.